### ISSUE TITLE
N:N Relationship Change Tracking

### ISSUE THREAD

**Issue by armandvanderwalt** [MicrosoftDocs/powerapps-docs#4733]
The documentation does not contain the details on how N:N change tracking works.

When enabling change tracking on both sides of the relationship change tracking is automatically enabled on the relationship entity

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ae183946-fb2a-1b2f-e199-273eb9a68d54
* Version Independent ID: 0cf78a90-f63b-af43-8075-c1cd4e934e7d
* Content: [Use change tracking to synchronize data with external systems (Microsoft Dataverse) - Power Apps](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/use-change-tracking-synchronize-data-external-systems)
* Content Source: [powerapps-docs/developer/data-platform/use-change-tracking-synchronize-data-external-systems.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/use-change-tracking-synchronize-data-external-systems.md)
* Service: **powerapps**
* Sub-service: **dataverse-developer**
* GitHub Login: @paulliew
* Microsoft Alias: **paulliew**

**Comment by JimDaly** [comment]
@armandvanderwalt 
Thanks for your feedback.

I just quickly reviewed the article and it says:

> To enable change tracking, set the [EntityMetadata.ChangeTrackingEnabled Property](https://learn.microsoft.com/en-us/dotnet/api/microsoft.xrm.sdk.metadata.entitymetadata.changetrackingenabled#microsoft-xrm-sdk-metadata-entitymetadata-changetrackingenabled) to `true`.

If I understand correctly, you are calling out that `ChangeTrackingEnabled` will be set to `true` on the intersect table ('relationship entity') automatically when the tables on both sides of the N:N relationship have `ChangeTrackingEnabled` set to `true`.

This sounds right, but I wouldn't have guessed it is important to know. How does this help you?

My expectation is that you would ignore intersect tables for change tracking and only focus on the tables that contain customer data. The intersect table is an implementation that enables the N:N relationship, but I wouldn't expect you would need to retrieve changes from that table.  I may be wrong about that.

Did you try to set `ChangeTrackingEnabled` directly on the intersect table?

**Comment by armandvanderwalt** [comment]
We have a use case were it was important for us to know about changes on the intersect table. and I stumbled on a stack overflow post that indicated once enabled on both sides it get's enabled on the intersect table also.

I did a metadata query that indicated the intersect table could not be enabled for change tracking so never attempted to set it directly.

Also something weird is that when doing a changes query the table name is appended with the word set at the end. Found this in the same metadata query
